src/components/SignIn/SignIn.jsx

```diff
@@ -33,6 +35,25 @@ export default function SignIn({
         clearInterval(timer);
         const accessToken = getAccessToken();
 
+        if (accessToken) {
+          if (Object.keys(tokenStoragePolicy).length !== 0) {
+            const {name, place} = tokenStoragePolicy;
+
+            switch (place) {
+              case 'cookie':
```

Comment:
if you see that there's a need for these strings to be used in multiple places now or in the future please move to some sort of constants and use it from there